### ISSUE TITLE
On wordpress 3.9 function pods_sanitize_like() calls $wpdb->esc_like() wich is introduced in WP 4.0

### ISSUE THREAD

**Issue by kruglikov** [pods-framework/pods#2606]
Got an error: 
Fatal error: Call to undefined method W3_Db::esc_like() in /plugins/pods/includes/data.php on line 121

Here is the code there:
```php

	else {
		global $wpdb;

		if ( pods_version_check( 'wp', '4.0', '<' ) ) {
			// like_escape is deprecated in WordPress 4.0
			$output = like_escape( pods_sanitize( $input ) );
		}
		else {
			$output = $wpdb->esc_like( pods_sanitize( $input ) );
		}
	}
```
Line 120:
```$output = $wpdb->esc_like( pods_sanitize( $input ) );```

Function pods_version_check( 'wp', '4.0', '<' ) returns false for WP 3.9.3. 
That's why line 120 runs, though I think it should not run and ```php
$output = like_escape( pods_sanitize( $input ) );
``` (line 117)should run instead.

**Comment by kruglikov** [comment]
Seems wrong usage of  pods_version_check( ) or error in this function.
Changed it like this in my local version and works fine:
```php

        global $wp_version;
		if ( version_compare( $wp_version, '4.0', '<' ) ) {
			// like_escape is deprecated in WordPress 4.0
			$output = like_escape( pods_sanitize( $input ) );
		}
		else {
			$output = $wpdb->esc_like( pods_sanitize( $input ) );
		}
```

**Comment by Shelob9** [comment]
This is a duplicate of #2559 which has already been fixed. It will be included in the 2.5.1 release, which should be available soon via WordPress.org. You can get the almost complete 2.5.1 by pulling the 2.x branch or from http://pods.io/latest
